### PR TITLE
feature/293-user-profile-update-user-profile-data-ui

### PR DESCRIPTION
**Pull request by superafroman**
No drop downs for profile details yet, but user journey implemented. Also includes a quick fix for logout (needs the latest wso2)

### CODE REVIEW

File: service/ui/page/profile/edit-success.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<Page title="Edit Profile Complete">
```
Comment (jenbutongit) [inline service/ui/page/profile/edit-success.html]: add :user here, so the sign out link displays properly.

Comment (superafroman) [reply service/ui/page/profile/edit-success.html]: Updated to make current user available globally so we don't have to pass around.

File: service/ui/component/Nav.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
 		</a>
 		<nav id="proposition-menu">
 			<ul id="proposition-links">
+				{{#if signedInUser}}
```
Comment (jenbutongit) [inline service/ui/component/Nav.html]: Now only the sign in link works, is signedInUser getting passed into this component for it to use?

Comment (jenbutongit) [inline service/ui/component/Nav.html]: Now only the sign in link works/ is visible, is signedInUser getting passed into this component for it to use?

**Pull request merged**